docs/building_models.md

```diff
@@ -188,6 +188,9 @@ The model id is of the form "model_name:model_version". Thus your command should
 python Inner/ML/runner.py --azureml --model=Prostate --cluster=my_cluster_name \
    --no-train --model_id=Prostate:1
 ```
+To evaluate the model on your own Azure ML Dataset, you can override the default dataset by adding the flag
+`--azure_dataset_id=<my dataset id>`. To specify multiple Datasets, you can additionally use the flag `--extra_azure_dataset_ids`
```

Comment:
I think a period is missing. Also, I think the last sentence is not clear. What else should happen when adding the flag? Maybe it's just easier to suggest running the script with `--help` for more information.

Reply:
We wanted to include a point about overriding the dataset specifically. I have tried to clarify the comment, hopefully it makes more sense now